documentation/guide-kafka.asciidoc

```diff
@@ -6,393 +6,7 @@ toc::[]
+For link:spring.asciidoc[spring], devonfw uses link:https://spring.io/projects/spring-kafka[Spring-Kafka] as kafka framework.
+For more details, check the link:spring/guide-kafka-spring.asciidoc[devon4j-kafka].
 
-This guide explains how Spring kafka is used in devonfw applications. It focuses on aspects which are special to devonfw if you want to learn about spring-kafka you should adhere to springs references documentation.
-
-There is an example of simple kafka implementation in the https://github.com/devonfw-sample/devon4j-kafka-employeeapp[devon4j-kafka-employeeapp].
-
-The devon4j-kafka library consist of:
-
-* Custom message processor with retry pattern
-* Monitoring support
-* Tracing support
-* Logging support
-* Configuration support for Kafka Producers, Consumers, brave tracer and message retry processing including defaults
-
-== How to use?
-
-To use devon4j-kafka you have to add required starter dependencies which is "starter-kafka-sender" or "starter-kafka-receiver" from devon4j. These 2 starters are responsible for taking care of the required spring configuration. If you only want to produce messages "starter-kafka-sender" is enough. For consuming messages you need "starter-kafka-receiver" which also includes "starter-kafka-sender".
-
-To use devon4j-kafka message sender add the below dependency:
-
-[source,xml]
-----
-<dependency>
-  <groupId>com.devonfw.java.starters</groupId>
-  <artifactId>devon4j-starter-kafka-sender</artifactId>
-</dependency>
-----
-
-It includes the Tracer implementations from Spring cloud sleuth.
-
-To use the devon4j-kafka message receiver configurations , loggers and message retry processor for processing message, add the below dependency:
-
-[source,xml]
-----
-<dependency>
-  <groupId>com.devonfw.java.starters</groupId>
-  <artifactId>devon4j-starter-kafka-receiver</artifactId>
-</dependency>
-----
-
-==  Property Parameters
-
-As written before kafka-producer and listener-specific configuration is done via properties classes. These classes provide useful defaults, at a minimum the following parameters have to be configured:
-
-[source,properties]
-----
-messaging.kafka.common.bootstrap-servers=kafka-broker:9092
-messaging.kafka.consumer.group-id=<e.g. application name>
-messaging.kafka.listener.container.concurrency=<Number of listener threads for each listener container>
-----
-
-All the configuration beans for devon4j-kafka are annotated with `@ConfigurationProperties` and use common prefixes to read the property values from `application.properties` or `application.yml`.
-
-Example:
-
-[source,java]
-----
- @Bean
-  @ConfigurationProperties(prefix = "messaging.kafka.producer")
-  public KafkaProducerProperties messageKafkaProducerProperties() {
-    return new KafkaProducerProperties();
-  }
-----
-
-For producer and consumer the prefixes are `messaging.kafka.producer...` and `message.kafka.consumer...`` and for retry the prefix is `messaging.retry...``
```

Comment:
One backtick too much after `message.kafka.consumer...` and `messaging.retry...`